lib/linguist/languages.yml

```diff
@@ -4111,6 +4111,8 @@ Python:
   - python2
   - python3
   aliases:
+  - bzl
+  - build
```

Comment:
Wouldn't it make more sense to add `starlark` as an alias instead?

`bzl` is fine, but I don't think we should be associating a word as generic as `build` with a specific language (unless that language is *literally* named "Build"...)

Reply:
Sounds good.